### PR TITLE
OS: RADWIN - Basic Support

### PR DESCRIPTION
**Pull request by InsaneSplash**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

### CODE REVIEW

**Comment by InsaneSplash** [comment]
This is my first OS PR so I might need some guidance. I included the MIB for future, and the over graphs are not used either... I'll attach some debug info to help?...

**Comment by InsaneSplash** [comment]
[poller-10.105.106.139.txt](https://github.com/librenms/librenms/files/721247/poller-10.105.106.139.txt)
[discovery-10.105.106.139.txt](https://github.com/librenms/librenms/files/721246/discovery-10.105.106.139.txt)
[radwin-10.105.106.139.txt](https://github.com/librenms/librenms/files/721248/radwin-10.105.106.139.txt)

**Comment by laf** [comment]
How come you closed this?

File: includes/definitions/radwin.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+os: radwin
```
Comment (laf) [inline includes/definitions/radwin.yaml]: Is this the best / correct name for the firmware / OS these devices run?

File: includes/definitions/radwin.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+text: 'RADWIN'
+type: wireless
+icon: radwin
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+register_mibs:
```
Comment (laf) [inline includes/definitions/radwin.yaml]: You can get rid of this unless you are trying to use mib polling.

File: tests/OSDiscoveryTest.php [hunk 3]
```diff
@@ -1222,6 +1222,11 @@ public function testRadlan()
         $this->checkOS('radlan');
     }
 
+    public function testRADWIN()
```
Comment (laf) [inline tests/OSDiscoveryTest.php]: Can you change this to testRadwin

File: html/images/os/radwin.svg [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (laf) [inline html/images/os/radwin.svg]: Can you compress this? https://jakearchibald.github.io/svgomg/

File: includes/definitions/radwin.yaml (hunk 1 above)
Comment (InsaneSplash) [inline includes/definitions/radwin.yaml]: Yeah, from what I can find. When you telnet in to the device, it just spits out a software revision.

Hello admin, welcome to ODU Management CLI!
+-----------------------------------------------------------+
     Software Revision          2.6.00_b2830_May  2 2011
+-----------------------------------------------------------+

File: includes/definitions/radwin.yaml (hunk 2 above)
Comment (InsaneSplash) [inline includes/definitions/radwin.yaml]: Ok cool, done.

File: html/images/os/radwin.svg (hunk 4 above)
Comment (InsaneSplash) [inline html/images/os/radwin.svg]: Ok  done :)

**Comment by InsaneSplash** [comment]
https://en.wikipedia.org/wiki/Radwin
http://www.radwin.com/

**Pull request merged**